test/images/agnhost/connect/connect.go

```diff
@@ -114,11 +120,24 @@ func connectSCTP(dest string, timeout time.Duration) {
 	errCh := make(chan error)
 
 	go func() {
-		conn, err := sctp.DialSCTP("sctp", nil, addr)
-		if err == nil {
-			conn.Close()
+		conn, err = sctp.DialSCTP("sctp", nil, addr)
+		if err != nil {
+			errCh <- err
+			return
+		}
+		defer func() {
+			errCh <- conn.Close()
+		}()
+
+		if _, err = conn.Write([]byte(fmt.Sprintf("%s\n", data))); err != nil {
+			errCh <- err
+			return
+		}
+
+		if _, err = conn.Read(buf); err != nil {
```

Comment:
do we have guarantee something is going to write back? I can't remember if we implemented that in the sctp server

Reply:
Yes, the sctp server replies back.

Reply:
Also, it tested this locally.